nservicebus/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -62,9 +62,9 @@ Each endpoint uses its own Azure storage account, thereby increasing message thr
 ![Scale out with multiple storage accounts](azure03.png "width=50%")
 
 
-## Securing connection strings to storage accounts for Scale Out
+## Using aliases for connection strings to storage accounts for Scale Out
 
-When the message throughput is increased, by using multiple storage accounts, its worth to consider using their names instead of raw connection strings. When applied, raw connection string values are replaced with registered names removing the possibility of leaking a connection string value. The concept of [securing connection strings to storage accounts](/nservicebus/azure-storage-queues/configuration.md#securing-connection-strings-to-storage-accounts) has been introduced in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Version 7. When using a single account, securing connection string is limited to calling `.UseAccountNamesInsteadOfConnectionStrings()`. When Scaling Out is applied, securing connection strings requires using logical names of the accounts.
+When the message throughput is increased, by using multiple storage accounts, it's worth to consider using aliases instead of raw connection strings. When applied, raw connection string values are replaced with registered aliases removing the possibility of leaking a connection string value. The concept of [using aliases for connection strings to storage accounts](/nservicebus/azure-storage-queues/configuration.md#using-aliases-for-connection-strings-to-storage-accounts) has been introduced in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Version 7. When using a single account, aliasing connection string is limited to calling `.UseAccountAliasesInsteadOfConnectionStrings()`. When Scaling Out is applied, it requires registering an alias for each storage account.
```

Comment:
Then how about simplifying it like this:

In order to prevent accidentally leaking connection string values, it's recommended to use [aliases instead of raw connection strings](/nservicebus/azure-storage-queues/configuration.md#using-aliases-for-connection-strings-to-storage-accounts).

An alias needs to be registered for every used storage account: